cmd/helm-operator/main.go

```diff
@@ -23,8 +23,8 @@ import (
 	clientset "github.com/fluxcd/helm-operator/pkg/client/clientset/versioned"
 	ifinformers "github.com/fluxcd/helm-operator/pkg/client/informers/externalversions"
 	"github.com/fluxcd/helm-operator/pkg/helm"
-	"github.com/fluxcd/helm-operator/pkg/helm/v2"
-	"github.com/fluxcd/helm-operator/pkg/helm/v3"
+	v2 "github.com/fluxcd/helm-operator/pkg/helm/v2"
+	v3 "github.com/fluxcd/helm-operator/pkg/helm/v3"
```

Comment:
The alias here is obsolete, and is probably caused by some enabled linter.

Reply:
Changed this.